### PR TITLE
Add jobs

### PR DESCRIPTION
**Pull request by mohitdutt**
get api and checkbox design

### CODE REVIEW

File: src/components/FilterScope.vue [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        return {
+            searchtext: '',
+            selectedSkils: [],
+            options: [
+                { text: 'React', value: 'React' },
+                { text: 'Angular', value: 'Angular' },
+                { text: 'Vue', value: 'Vue' },
+                { text: 'Web Components', value: 'WebComponents' }
+            ],
+            fulltimestatus: false, parttimestatus: false, remotestatus: false, Contractstatus: false,
```
Comment (souvikbasu) [inline src/components/FilterScope.vue:66]: camelCase variable names

File: src/components/FilterScope.vue [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+            selectedlevels: '',
+            searchFilter: []
+        }
+    },
+    mounted() {debugger
```
Comment (souvikbasu) [inline src/components/FilterScope.vue:71]: No debugger statement in prod

File: src/components/FilterScope.vue [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+        for(let i=0; i< this.$el.getElementsByClassName('skillscheckbox')[0].children.length; i++){
```
Comment (souvikbasu) [inline src/components/FilterScope.vue:72]: Do not access Dom elements by code. Use data binding

File: src/components/FilterScope.vue [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+            this.$el.getElementsByClassName('skillscheckbox')[0].children[i].style.display = '-webkit-box'
+            this.$el.getElementsByClassName('skillscheckbox')[0].children[i].style.marginTop = '1rem'
```
Comment (souvikbasu) [inline src/components/FilterScope.vue:74]: Do not set style by direct access to Dom. Set style the Vue way :)

File: src/components/FilterScope.vue [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+        }
+    },
+    methods: {
+        onChangesSearch(){debugger
```
Comment (souvikbasu) [inline src/components/FilterScope.vue:78]: Remove all debugger

File: src/components/FilterScope.vue [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+            this.Contractstatus = x
+            this.getFilterResults()
+        },
+        onChangeslevels(x){
+            this.selectedlevels = x
+            this.getFilterResults()
+        },
+
+        getFilterResults(){debugger
+            Request.getData('job?searchText='+this.searchtext+'&skills='+this.selectedSkils.toString()+'&isFullTime='+this.fulltimestatus+'&isPartTime='+this.parttimestatus+'&isRemote='+this.remotestatus+'&isContract='+this.Contractstatus+'&level='+this.selectedlevels+'').then((response)=>{
```
Comment (souvikbasu) [inline src/components/FilterScope.vue:107]: Api call code should ideally move to services class. Not in presentation classes

File: src/components/JobList.vue [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+        
+        if(this.joblist.description.length>630){
+            var showChar = 630;
+            var ellipsestext = ". . . . .";
+            var moretext = "Click for more description";
+            var lesstext = "Click for less description"; 
+            var showcontent = this.joblist.description.substr(0, showChar);
+            var hiddencontent = this.joblist.description.substr(showChar-1, this.joblist.description.length - showChar);
+            var html = showcontent + '<span class="moreellipses">' + ellipsestext+'</span><span class="morecontent"><span style="display: none;">' + hiddencontent + '</span><br><p class="morelink">' + moretext + '</p><p class="lesslink" style="display:none">' + lesstext + '</p></span>';
+            this.$el.getElementsByClassName('jobdescription')[0].innerHTML = html
```
Comment (souvikbasu) [inline src/components/JobList.vue:46]: Don't set innerhtml explicitly. Use binding

File: src/components/JobList.vue [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+        }
+        
+        if(this.joblist.description.length>630){
+            this.$el.getElementsByClassName('jobdescription')[0].getElementsByClassName('morelink')[0].addEventListener('click', function(event) {debugger
```
Comment (souvikbasu) [inline src/components/JobList.vue:50]: Such a long statement!!

**Review by souvikbasu — CHANGES_REQUESTED** [review]

File: src/components/JobList.vue (hunk 8 above)
Comment (mohitdutt) [reply src/components/JobList.vue:50]: i am not getting this part.

Comment (mohitdutt) [reply src/components/JobList.vue]: i am not getting this part.

File: src/components/FilterScope.vue (hunk 6 above)
Comment (mohitdutt) [reply src/components/FilterScope.vue:107]: Sir, can you please explain this one. as i used api like this in adsjobs page also.

**Pull request closed without merging**